.gitignore

```diff
@@ -1,6 +1,7 @@
 .DS_Store
 .vscode
 node_modules
+!action/node_modules
```

Comment:
to avoid having to check in the `node_modules` folder, I think it might be worth looking at adding a build step that uses [ncc](https://help.github.com/en/actions/creating-actions/creating-a-javascript-action).